src/redux/features/project.slice.ts

```diff
@@ -4,23 +4,24 @@ import { Project } from "../../utils/interfaces";
 import mockData from "../../api/mockData.json";
 
 interface StoreState {
-  project: Project | null;
+  project: Project;
```

Comment:
if you want to put the project props directly in the store, then there is no need for StoreState interface
+
I believe the project should be able to be NULL, like maybe later in someplace we dont want to allow a certain procedure to happen if no Project was selected ( project is null )
but this way, it is always set to something

Reply:
yep that makes sense 🙏 nice catch